prov/efa/src/rdm/efa_rdm_pke.c

```diff
@@ -390,17 +390,18 @@ ssize_t efa_rdm_pke_sendv(struct efa_rdm_pke **pkt_entry_vec,
 	struct efa_conn *conn;
 	struct efa_rdm_ep *ep;
 	struct efa_rdm_pke *pkt_entry;
-	struct efa_rdm_peer *peer;
+	struct efa_rdm_ope *txe;
 	struct ibv_sge sg_list[2];  /* efa device support up to 2 iov */
 	struct ibv_data_buf inline_data_list[2];
 	int ret, pkt_idx, iov_cnt;
 
 	assert(pkt_entry_cnt);
 	ep = pkt_entry_vec[0]->ep;
 	assert(ep);
-	peer = efa_rdm_ep_get_peer(ep, pkt_entry_vec[0]->addr);
```

Comment:
You only need peer instead of txe in this function, why not just assign peer as txe->peer once so you don't have to reference it as `txe->peer` everywhere in the function

Reply:
done